config/locales/client.en.yml

```diff
@@ -1353,6 +1353,7 @@ en:
       saved_local_draft_tip: "saved locally"
       similar_topics: "Your topic is similar to..."
       drafts_offline: "drafts offline"
+      edit_conflict: '<i class="fa fa-exclamation-triangle"></i> edit conflict'
```

Comment:
I wonder if this will be broken by the FA5 change @pmusaraj? Maybe the icon should be added in the handlebars template rather than the translation string?

Reply:
Yes, the FA5 change would break this. Best to put the icon in the handlebars template.

Reply:
It looks like there are 3 more icons left in i18n files.

Reply:
Yes, thanks, I've addressed those in the fontawesome5 branch.